app/controllers/users_controller.rb

```diff
@@ -8,6 +8,24 @@ def create
     @user = User.new(user_params)
     if @user.save
       process_invitation
+      # Set your secret key: remember to change this to your live secret key in production
```

Comment:
I would prefer to delete these comments. You will be moving the code anyway in next lessons.